### PR TITLE
DRY the deploy jobs and remove ma:release

### PR DESCRIPTION
**Pull request by weitzman**
Also removes ma:release command as it no longer works.

**Jira:**
https://jira.mass.gov/browse/DP-16585

**To Test:**
- [ ] Code review before merge
- [ ] After merge, lets do a release and hotfix, without taking those to Prod. That will exercise some workflows.

---

[Peer Review Checklist](https://github.com/massgov/opemass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -134,7 +134,7 @@ jobs:
     <<: *container_config
     steps:
       - attach_workspace: {at: /var/www}
-      - run: yarn danger ci
+      - run: yarn danger ci --failOnErrors
```
Comment (weitzman) [inline .circleci/config.yml:137]: Now that Danger is its own job we want that job to fail if danger fails.

File: .circleci/config.yml [hunk 2]
```diff
@@ -499,6 +430,8 @@ workflows:
           requires:
             - build
             - push_acquia
+          refreshDb: "1"
```
Comment (mrossi113) [inline .circleci/config.yml]: Nothing big, but does the order in parameter matter?
I'm seeing a different order in multiple places.

File: .circleci/config.yml [hunk 3]
```diff
@@ -499,6 +430,8 @@ workflows:
...
+          target: cd
```
Comment (mrossi113) [inline .circleci/config.yml:434]: Nothing big, but does the order in parameter matter?
I'm seeing a different order in multiple places.

File: .circleci/config.yml [hunk 4]
```diff
@@ -555,21 +488,23 @@ workflows:
...
+          requires: [deploy]
           target: test
           samples: 700
           filters:
             branches:
               only: /^release\/.*$/
       - backstop:
-          requires: [deploy_release_stage]
+          requires: [deploy]
           target: stage
```
Comment (mrossi113) [inline .circleci/config.yml]: I just noticed the target for backstop should be `test` missed in my first check.

File: .circleci/config.yml [hunk 5]
```diff
@@ -599,21 +534,22 @@ workflows:
...
+          requires: [deploy]
           target: test
           samples: 200
           filters:
             branches:
               only: /^hotfix\/.*$/
     - backstop:
-          requires: [deploy_hotfix_stage]
+          requires: [deploy]
           target: stage
```
Comment (mrossi113) [inline .circleci/config.yml]: Same here the backstop should have `test` not stage as target.

File: .circleci/config.yml [hunk 6]
```diff
@@ -643,21 +579,23 @@ workflows:
...
+          requires: [deploy]
           target: test
           samples: 500
           filters:
             branches:
               only: /^mayflower\/.*$/
       - backstop:
-          requires: [deploy_release_stage]
+          requires: [deploy]
           target: stage
```
Comment (mrossi113) [inline .circleci/config.yml]: Same here change target to `test`

**Review by mrossi113 — CHANGES_REQUESTED** [review]
@weitzman sorry for the delay, but I had a question and noticed backstop is using the wrong target for stage environment we use `test`. Similar to the nightcrawler command and deploy.

File: .circleci/config.yml (hunk 3 above)
Comment (weitzman) [reply .circleci/config.yml:434]: Order doesnt matter, fortunately.

**Review by mrossi113 — APPROVED** [review]
👍 LGTM!

**Pull request merged**